Article/Article.js

```diff
@@ -26,4 +26,8 @@ class Article {
 
 */
 
-let articles;
\ No newline at end of file
+let articles = document.querySelectorAll(".article");
+
+articles = Array.from(articles).map(function(item) {
```

Comment:
It seems you haven't started the expand button functi0on on this component. There is quite a bit that needs to be done on this file so I would reference the solution code.